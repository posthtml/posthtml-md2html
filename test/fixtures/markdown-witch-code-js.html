<div md>
```js
  function foo(bar) {
    var a = 42,
        b = 'Prism';
    return a + bar(b);
  }
```
</div>