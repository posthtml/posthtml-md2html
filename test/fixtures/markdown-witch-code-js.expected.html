<div><pre><code class="language-js">  function foo(bar) {
    var a = 42,
        b = &#39;Prism&#39;;
    return a + bar(b);
  }</code></pre>
</div>